src/main/java/org/isf/patient/gui/PatientInsert.java

```diff
@@ -181,39 +173,33 @@ public void actionPerformed(ActionEvent actionEvent) {
 	 */
 	public PatientInsert(JDialog owner, Patient old, boolean inserting) {
 		super(owner, true);
-		patient=old;
-		insert=inserting;
+		patient = old;
+		insert = inserting;
 		initialize();
 	}
-	
+
 	public PatientInsert(JFrame owner, Patient old, boolean inserting) {
 		super(owner, true);
-		patient=old;
-		insert=inserting;
+		patient = old;
+		insert = inserting;
 		initialize();
 	}
 	
 	/**
 	 * This method initializes this
 	 */
 	private void initialize() {
-		
-		
 		Toolkit kit = Toolkit.getDefaultToolkit();
 		Dimension screensize = kit.getScreenSize();
-		pfrmBordX = (screensize.width - (screensize.width / pfrmBase * pfrmWidth)) / 
-		2;
-		pfrmBordY = (screensize.height - (screensize.height / pfrmBase * 
-		pfrmHeight)) / 2;
-		this.setBounds(pfrmBordX+10,pfrmBordY+10,screensize.width / pfrmBase * 
-		pfrmWidth,screensize.height / pfrmBase * pfrmHeight);
+		pfrmBordX = (screensize.width - (screensize.width / pfrmBase * pfrmWidth)) / 2;
+		pfrmBordY = (screensize.height - (screensize.height / pfrmBase * pfrmHeight)) / 2;
+		this.setBounds(pfrmBordX + 10, pfrmBordY + 10, screensize.width / pfrmBase * pfrmWidth, screensize.height / pfrmBase * pfrmHeight);
 		this.setContentPane(getJContainPanel());
 		this.setTitle(MessageBundle.getMessage("angal.patient.newpatient.title"));
-		this.setSize(new java.awt.Dimension(604,445));
+		this.setSize(new java.awt.Dimension(604, 445));
 		setSize(screensize.width / pfrmBase * pfrmWidth,
 				screensize.height / pfrmBase * pfrmHeight);
```

Comment:
Could it be replaced by pack() + setLocationRelativeTo(null)?

Reply:
Done

Reply:
Thanks!